Dockerfile

```diff
@@ -0,0 +1,133 @@
+# (Keep the version in sync with the node install below)
+FROM node:18-bullseye-slim as frontend
+
+# Make build & post-install scripts behave as if we were in a CI environment (e.g. for logging verbosity purposes).
+ARG CI=true
+
+WORKDIR /app
+
+# Install front-end dependencies.
+# This will create a `node_modules` directory in the current directory.
+COPY package.json package-lock.json tailwind.config.js esbuild.config.js contribute.json ./
+COPY ./tailwind-plugins/ ./tailwind-plugins/
+RUN npm ci --no-optional --no-audit --progress=false
+
+# Compile static files from static source at ./source to ./network-api/networkapi/frontend
+# This will create a `network-api/networkapi/frontend` directory.
+COPY ./source/ ./source/
+COPY ./network-api/networkapi/ ./network-api/networkapi/
+RUN npm run build
+
+
+# We use Debian images because they are considered more stable than the alpine
+# ones becase they use a different C compiler. Debian images also come with
+# all useful packages required for image manipulation out of the box. They
+# however weight a lot, approx. up to 1.5GiB per built image.
+#
+# Note: This stage builds the base image for production. Presently we are not
+# using this on the production site, but only use it as base for the dev build.
+FROM python:3.9.9-slim as base
+
+# Install dependencies in a virtualenv
+ENV VIRTUAL_ENV=/app/dockerpythonvenv
+
+RUN useradd mozilla --create-home && mkdir /app $VIRTUAL_ENV && chown -R mozilla /app $VIRTUAL_ENV
+
+WORKDIR /app
+
+# Set default environment variables. They are used at build time and runtime.
+# If you specify your own environment variables on Heroku or Dokku, they will
+# override the ones set here. The ones below serve as sane defaults only.
+#  * PATH - Make sure that Poetry is on the PATH, along with our venv
```

Comment:
We might want to remove "Poetry" here, as we are not using it on Moz. Dependencies are managed through `pip-tools`.

Reply:
Ah you got me. I copied that from Wagtail Kit. Will remove.